literature.bib

```diff
@@ -6657,6 +6916,24 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{GZH+:ECCV16,
+  title        = {{MS-Celeb-1M: A Dataset and Benchmark for Large-Scale Face Recognition}},
+  author       = {Guo, Yandong and Zhang, Lei and Hu, Yuxiao and He, Xiaodong and Gao, Jianfeng},
+  booktitle    = {Computer Vision -- ECCV 2016},
```

Comment:
This conference name looks messed up. ECCV is an abbreviation for a longer title (european conference on computer vision) as long as I remember from my old days at ICG.